package.json

```diff
@@ -7,12 +7,13 @@
     ".prettierrc.json",
     ".editorconfig",
     ".gitattributes",
-    "bin/*.js"
+    "bin/*.js",
+    ".husky/*"
   ],
   "bin": {
-    "run-e": "./bin/run_e.js",
-    "run-ci": "./bin/run_ci.js",
-    "run-local": "./bin/run_local.js"
```

Comment:
In most of our repositories, we use `exports`, which requires file paths to start with `./`. While removing in `bin` is possible, we might want to keep it for consistency. Also, this change might be unrelated to Husky?

Reply:
This is done automatically by `npm init -y`. I can revert it, but I'd trust npm more here :)

But yeah it's unrelated to the husky update, I just ran `npm init -y` in this branch by mistake.

Reply:
`npm init` should not be needed anymore for this repository, since it's already been initialized, so this should not be a problem for contributors?

Reply:
I reverted the change already.

Reply:
Thanks!